python/lsst/ts/cbp/component.py

```diff
@@ -213,7 +213,9 @@ async def move_azimuth(self, position: float):
 
         """
         if position < -45 or position > 45:
-            raise ValueError("New azimuth value exceeds Azimuth limit.")
+            raise ValueError(
+                f"New azimuth value {position:0.2f} exceeds Azimuth limit [-45, 45]."
+            )
```

Comment:
Also, please consider a function that takes the value, the limits and a brief description of the value. That avoids the danger that the numerical value of a limit and the reported value can differ if somebody changes one and forgets to change the other, and simplifies the code a bit. e.g.:
```
def assert_in_range(name, value, min_value, max_value):
    """Raise ValueError if a value is out of range."""
    if value < min_value or value > max_value:
        raise ValueError(f"{name} = {value} not in range [{min_value}, {max_value}]")
```
(if you avoid the format then it can be used for integers as well).